docs/ota-client-guide/modules/ROOT/pages/yocto-release-branches.adoc

```diff
@@ -0,0 +1,22 @@
+= Yocto release branches
+
+== Supported branches
+
+Yocto has a number of release branches. Their details are documented in the https://wiki.yoctoproject.org/wiki/Releases[Yocto wiki]. HERE OTA Connect generally supports the most recent release branches. This current list is available in the https://github.com/advancedtelematic/meta-updater/blob/master/CONTRIBUTING.adoc[meta-updater documentation].
+
+== Changing branches
+
+Changing the release branch of an existing Yocto build environment or the software running on a device in production is something that should only be done with careful consideration. If you change a release branch in your build environment, you will mostly like need to re-bitbake every package for your image, which can take several hours. If you then try to update a device from a version built with an older branch to a version built with a newer branch, the size of the update will mostly like be quite large, which runs the risk of the device running out of disk space to download the update. There may also be unanticipated changes to other packages or the bootloader that could cause unexpected problems.
+
+That said, in general such changes are possible. However, due to a change in the bootloader configuration, upgrading from rocko (or earlier branches) to sumo (or later branches) is not supported.
```

Comment:
I think the breaking point is between sumo / thud when we started including initramfs in fit images.

Reply:
Ah, sorry, I misremembered and I seem to keep forgetting about how this came about. I didn't think there was much in sumo that wasn't in thud, though, so it is a bit surprising. Or did we intentionally keep that out of rocko because we didn't want to break things?

Reply:
@lbonn You're talking about https://github.com/advancedtelematic/meta-updater/pull/368, right? You are right, then: it was never backported, so it isn't in rocko or sumo. It is in thud and warrior since they were released after that PR was merged.

Reply:
yes indeed. Thanks for digging it out!